kod/object/item/passitem/numbitem/food/mug.kod

```diff
@@ -56,6 +57,8 @@ properties:
    viNutrition = 6
    piNumber = 1
 
+   piItem_flags = 57   % Slight golden hue to set it apart from mug of stout
```

Comment:
Please use the symbolic names from blakston.khd rather than hard-coding the values.